helpers/matrix.yml

```diff
@@ -13,4 +13,6 @@ KIBANA_SUITE:
   - 7.0.x
   - 5.x
 KUBERNETES_VERSION:
+  - '1.10'
```

Comment:
I actually removed this recently https://github.com/elastic/helm-charts/pull/65/files as GKE no longer has it available.

Reply:
It seems like they added it back in again or it was temporarily unavailable. As proof you can see it is passing in this pull request testing job https://devops-ci.elastic.co/job/elastic+helm-charts+pull-request+cluster-creation/93/ .

Reply:
Hmm interesting.

Reply:
https://cloud.google.com/kubernetes-engine/docs/release-notes#version-updates

> 1.10.12-gke.7 - This version is being made available again after being previously removed.
> 1.10.12-gke.9

Pretty weird really /shrug